backends/arm/_passes/fuse_batchnorm2d_pass.py

```diff
@@ -6,10 +6,15 @@
 # pyre-unsafe
 
 import torch
+from executorch.backends.transforms.utils import (
```

Comment:
This might need a buck file update. 
I.e. add `//executorch/backends/transforms:utils` in the deps list.